web/components/portfolio/portfolio-value-section.tsx

```diff
@@ -50,24 +55,22 @@ export const PortfolioValueSection = memo(
               </div>
             </Col>
           </div>
-          {!disableSelector && (
-            <select
-              className="select select-bordered self-start"
-              value={portfolioPeriod}
-              onChange={(e) => {
-                setPortfolioPeriod(e.target.value as Period)
-              }}
-            >
-              <option value="allTime">{allTimeLabel}</option>
-              <option value="weekly">Last 7d</option>
-              {/* Note: 'daily' seems to be broken? */}
-              {/* <option value="daily">Last 24h</option> */}
-            </select>
-          )}
+          <select
+            className="select select-bordered self-start"
+            value={portfolioPeriod}
+            onChange={(e) => {
+              setPortfolioPeriod(e.target.value as Period)
+            }}
+          >
+            <option value="allTime">All time</option>
+            <option value="weekly">Last 7d</option>
+            {/* Note: 'daily' seems to be broken? */}
+            {/* <option value="daily">Last 24h</option> */}
```

Comment:
Did you mean to enable this option? I think it's a good idea.

Reply:
Oops, just forgot to push that commit.